Nodejs/Product/Nodejs/Snippets/1033/Nodejs/alert.snippet

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>alert</Title>
+      <Description>Code Snippet for an Alert Message</Description>
+      <Shortcut>alert</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>message</ID>
+          <Default>message</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[alert($message$);]]>

```

Comment:
alerts are really only useful for browser-side code (there's no such concept in node because there's no front-facing component), so you can probably go ahead and remove this snippet since browser-side code is best dealt with in the  default VS editor. Especially since if we're getting to browsers, then we're opening up a whole new can of worms.